src/coreclr/tests/src/profiler/unittest/getruntimeinformation.cs

```diff
@@ -0,0 +1,38 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.IO;
+using System.Runtime.Loader;
+
+namespace Profiler.Tests
+{
+    class Program
+    {
+        static readonly Guid GetRuntimeInformationProfilerGuid = new Guid("4CF56B6D-F8FB-4056-AF4A-F6413DD738B1");
+
+        static int Main(string[] args)
+        {
+            if (args.Length == 1 && args[0].Equals("RunTest", StringComparison.OrdinalIgnoreCase))
+            {
+                return RunTest();
+            }
+
+            return ProfilerTestRunner.Run(profileePath: System.Reflection.Assembly.GetExecutingAssembly().Location,
```

Comment:
Can we set FX_PRODUCT_VERSION to an explicit value and then confirm it round-tripped through the test?

Reply:
It turns out no, because Configuration::GetKnobStringValue doesn't check environment variables. It only checks knobs specifically set through the hosting api. I think I'm going to have to give up trying to have a test for this, and just test it manually.